oneanddone/tests/functional/pages/home.py

```diff
@@ -18,6 +19,7 @@ class HomePage(Base):
     _suggested_first_tasks_heading_locator = (By.CSS_SELECTOR, '.task-list-container h3')
     _suggested_first_task_locator = (By.CSS_SELECTOR, '.task-list > li')
     _available_tasks_locator = (By.ID, 'available-tasks')
+    _find_search_button = (By.CSS_SELECTOR, '#id_search')
```

Comment:
Add a `_locator` suffix. Also, this could be a `By.ID`, but there's no need to update this.